examples/rp2/pio_quadrature_encoder.py

```diff
@@ -0,0 +1,100 @@
+"""
+This implementation offloads the quadrature sensor continuous pooling to the StateMachine pio_quadrature,
+pio_quadrature will detect state changes and push the values then raises an interrupt that is handled by
+encoder_state_changed_irq_handler
+
+encoder_state_changed_irq_handler was inspired from 
+https://electronics.stackexchange.com/questions/360637/quadrature-encoder-most-efficient-software-implementation
+
+"""
+
+from machine import Pin
+import rp2
+from rp2 import PIO, StateMachine
+import utime
+import array
+
+
+state_look_up_table = array.array("b", [
```

Comment:
Why not use a bytearray here?

Reply:
Using bytearray has strange effects on speed overrun, causing the count to make big leaps.

Reply:
That's interesting. Do you have any idea why this is?